docsets/shapely/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "shapely",
-    "version": "1.5.15",
+    "version": "1.6.4",
     "archive": "shapely.tgz",
     "author": {
         "name": "Sean Gillies",
         "link": "https://github.com/sgillies"
     },
     "aliases": [],
+		"specific_versions": [
+			{
+				"version": "1.6.4",
+				"archive": "versions/1.6.4/shapely.tgz"
+			},
+			{
+				"version": "1.5.15",
+				"archive": "versions/1.5.15/shapely.tgz"
```

Comment:
You need to add the archive at this path as well.

Reply:
Thanks! Now added.